pkg/util/conditions/condition_types.go

```diff
@@ -14,6 +14,8 @@ const (
 	ReconcileErrorReason      ConditionReason = "ReconcileError"
 	DeletingReason            ConditionReason = "Deleting"
 	DeletionFailedReason      ConditionReason = "DeletionFailed"
+	GarmAPIErrorReason        ConditionReason = "GarmAPIError"
+	ComponentNotReadyReason   ConditionReason = "ComponentNotReady"
```

Comment:
`ComponentNotReady` is something i'm currently struggling with. 
All the other errors are self-explained, but `ComponentNotReady` doesn't give me that much information - even with my current knowledge of pools, orgs, etc, ...

Reply:
fixed with [ecdb745](https://github.com/mercedes-benz/garm-operator/pull/104/commits/ecdb7456b50799937c024916029d0162f01a33d1)